plan-patches/cf-azure/ops/rename-network-and-deployment.yml

```diff
@@ -0,0 +1,252 @@
+
+- type: replace
+  path: /instance_groups/name=database/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=diego-api/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=uaa/networks/name=default/name
+  value: ((network_name))
+
+# - type: replace
+#   path: /instance_groups/name=singleton-blobstore/networks/name=default/name
+#   value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=api/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=cc-worker/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=router/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=scheduler/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=diego-cell/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=log-api/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=tcp-router/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=credhub/networks/name=default/name
+  value: ((network_name))
+
+- type: replace
+  path: /instance_groups/name=rotate-cc-database-key/networks/name=default/name
+  value: ((network_name))
+
+# Deployment Name
+- type: replace
+  path: /name
+  value: ((deployment_name))
+
+# Bosh DNS Aliases
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases
+  value:
+  - domain: '_.cell.service.cf.internal'
+    targets:
+    - query: '_'
+      instance_group: diego-cell
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+    - query: '_'
+      instance_group: windows-cell
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+    - query: '_'
+      instance_group: windows2016-cell
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+    - query: '_'
+      instance_group: isolated-diego-cell
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+  - domain: auctioneer.service.cf.internal
+    targets:
+    - query: '*'
+      instance_group: scheduler
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+  - domain: bbs.service.cf.internal
+    targets:
+    - query: 'q-s4'
+      instance_group: diego-api
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+  - domain: bits.service.cf.internal
+    targets:
+    - query: '*'
+      instance_group: bits
+      deployment: ((deployment_name))
+      network: ((network_name))
+      domain: bosh
+  # - domain: blobstore.service.cf.internal
```

Comment:
Did you mean to leave these comments as an example aliasing a blobstore? Or can we remove them?

Reply:
removed the commented out lines. thanks.